tests/test_main.py

```diff
@@ -58,3 +61,15 @@ def test_format_output_auto_expand():
                                      max_width=1)
     expanded = ['Title', u'-[ RECORD 0 ]-------------------------\nhead1 | abc\nhead2 | def\n', 'test status']
     assert expanded_results == expanded
+
+
+@dbtest
+def test_i_works(tmpdir, executor):
+    sqlfile = tmpdir.join("test.sql")
+    sqlfile.write("SELECT NOW()")
+    cli = PGCli(
+        pgexecute=executor,
+        pgclirc_file=config_location(),
```

Comment:
I found out why it is failing. 

This config_location is a directory under ~/.config/pgcli, but we create that directory (if it doesn't exist) when pgcli is launched. https://github.com/dbcli/pgcli/blob/master/pgcli/main.py#L611

Since we're not launching pgcli, that directory is never created. 

The easy fix will be to call os.makedirs() in this test, but the better fix will be to do that os.makedirs() in the PGCli's __init__() method. Here: https://github.com/dbcli/pgcli/blob/master/pgcli/main.py#L92

Would you like to make this change?